Governance/concepts/authorship.md

```diff
@@ -0,0 +1,51 @@
+# Authorship
+
+Research output are usually the result of collaboration, so it is important to attribute the right roles to anyone involved.
+Often researchers don't distinguish between different levels of involvements and either add only the main authors, failing to recognise minor but still important contributions to the work, or they add anyone involved as an author.
+However, `author` is not the only possible option, most data portals and publishing standards now allow to use `contributor` to cover a range of different roles.
+ 
+```{note}
+Depending on the standard 
+```
+While only the authors will be included in the research output citation, the contributors will still feature prominently in the record metadata and webpage, and it is often possible for them to use their researcher-id to keep track of their contributions.
+This is particularly important for data curators who often don't appear as authors but can now be recognised for their role. This practice it is also useful to a user of the research output as it makes clear who is more likely to answer a specific question.
+ 
+Different potential roles are described in this book [technical resources section](../tech/contributors), an `author` in brief is `a person who has created or modified a research output in a substantial way`. 
+
+## Authorship policy
+
+This is not just a definition, it is part of the requirements set by the [Australian Code for the Responsible Conduct of Research 2018](https://www.nhmrc.gov.au/about-us/publications/australian-code-responsible-conduct-research-2018#download), and as such, it is really important to be careful when publishing not to breach this rule.
+A complete definition is available in the code itself; usually it is also possible to refer to the Authorship Policy of the research output publisher.
+An Authorship Policy helps a contributor to make the right decision when defining the authors and helps the managers to solve any potential dispute. It should include a clear definition on author and contributors, including their responsibilities and what action will be taken in case of a breach. 
+
+```{tip}
+A Task Force of the [FORCE11 Software Citation Implementation Working Group](https://github.com/force11/force11-sciwg) has published a useful guide on best practices for software portals: [Nine Best Practices for Research Software Registries and Repositories: A Concise Guide](https://arxiv.org/pdf/2012.13117.pdf). This includes a list of policies that should be established to help managing the portal. Some of these recommendations can be adopted also for smaller data or software collections.  
+```
```

Comment:
With this {tip} and the following {note}, the URL hyperlinks do not work under these blocked out sections, but appear as typed. 
They also format as a single line that you need to scroll horizontally on, which makes them quite unwieldy.

Reply:
ok, I'll double checked this too, I thought it worked in mine so it might be a case of a missing library, otherwise I'll change them

Reply:
They work for me, so it might be a matter of missing extensions in the conda env. When you say they don't work I assume that you tried to build it locally. I would leave it until we merge and if also the main branch has the same issue, I will check which specific extension is needed.

Reply:
@chloemackallah I pushed the changes if you want to check again and let me know if the formatting is better

Reply:
Looks better. The URLs still aren't linking properly, but perhaps they will once merged and formatted properly in the book rather than just 'viewed' in the PR.

Reply:
URLs work in the book :-)